### PR TITLE
fix(css): fill-opacity should accept percentages

### PR DESCRIPTION
**Pull request by pamelalozano16**
### Description
`fill-opacity` should accept percentages (https://developer.mozilla.org/en-US/docs/Web/CSS/fill-opacity)

### CODE REVIEW

File: css/properties.json [hunk 1]
```diff
@@ -4766,7 +4766,7 @@
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/fill"
   },
   "fill-opacity": {
-    "syntax": "<alpha-value>",
+    "syntax": "<number> | <percentage>",
```
Comment (skyclouds2001) [inline css/properties.json:4769]: maybe [<'opacity'>](https://developer.mozilla.org/en-US/docs/Web/CSS/opacity) or <opacity-value> is better?

see https://github.com/mdn/data/pull/784

File: css/properties.json [hunk 2]
```diff
@@ -4766,7 +4766,7 @@
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/fill"
   },
   "fill-opacity": {
-    "syntax": "<alpha-value>",
+    "syntax": "<opacity>",
```
Comment (skyclouds2001) [inline css/properties.json:4769]:
```suggestion
    "syntax": "<'opacity'>",
```

note: `<opacity>` is not equal to `<'opacity'>`

https://github.com/mdn/data/blob/8ec6d5057d8b774c185e33fe684b5af4dc8f3333/css/properties.json#L9576-L9591

Comment (pamelalozano16) [reply css/properties.json]: Thanks!

**Review by pepelsbey — APPROVED** [review]
Looks good! Thank you :)

**Pull request merged**